packages/docs/getting-started/running-attestation-service.md

```diff
@@ -0,0 +1,125 @@
+
+### Nexmo
+
+Here is the list of the enviromnet variables needed to use the Nexmo SMS broker:
+
+| Variable        | Explanation                                                     |
+| --------------- | --------------------------------------------------------------- |
+| NEXMO_KEY       | The API key to the Nexmo API                                    |
+| NEXMO_SECRET    | The API secret to the Nexmo API                                 |
+| NEXMO_BLACKLIST | A comma-sperated list of country codes you do not want to serve |
+
+### Twilio
+
+If you prefer using Twilio, this is list of the variables to use:
+
+| Variable                     | Explanation                                                     |
+| ---------------------------- | --------------------------------------------------------------- |
+| TWILIO_ACCOUNT_SID           | The Twilio account ID                                           |
+| TWILIO_MESSAGING_SERVICE_SID | The Twilio Message Service ID. Starts by `MG`                   |
+| TWILIO_AUTH_TOKEN            | The API authentication token                                    |
+| TWILIO_BLACKLIST             | A comma-sperated list of country codes you do not want to serve |
+
+## Accounts Configuration
+
+First we need to create an account for getting the attestation key needed to sign the attestations. Run:
+
+```bash
+celocli account:new
+```
+
+We copy the account details and assign the Private Key to the `ATTESTATION_KEY` environment variable:
+
+```bash
+export ATTESTATION_KEY=<Private Key>
+```
+
+The Attestation Service needs to connect to a Web3 Provider. This is going to depend on the network you want to connect. So depending on which network you are making available the service, you need to configure the `CELO_PROVIDER` variable pointing to that.
+
+For example:
+
+```bash
+# Web3 provider for Alfajores network
+export CELO_PROVIDER="https://alfajores-forno.celo-testnet.org/"
+```
+
+#### Database Configuration
+
+For storing and retrieving the attestation requests the service needs a database to persist that information. Currently `sqlite`, `postgres` and `mysql` are supported. For testing purposes you can use `sqlite` but it's recommended to run a stand-alone database server using `mysql` or `postgres` if your intention is running the Attestation Service in a production environment.
+
+So for specifying the database url you need to setup the `DATABASE_URL` variable:
+
+```bash
+export DATABASE_URL="sqlite://db/dev.db"
+export DATABASE_URL="mysql://user:password@mysql.example.com:3306/attestation-service"
+export DATABASE_URL="postgres://user:password@postgres.example.com:5432/attestation-service"
+```
+
+You can find the migration scripts for creating the schema at the `celo-monorepo`, `packages/attestation-service` folder. From there, after setting up the `DATABASE_URL` env variable you can run the following commands:
+
+```bash
+yarn run db:create
```

Comment:
This won't work other than the `sqlite` driver

Reply:
removing that `db:create` line and adding reference that depending on the database technology the users need to create the database with the proper user access.